app/uk/gov/hmrc/traderservices/views/EnterResponseTextView.scala.html

```diff
@@ -31,14 +31,27 @@
     <div class="govuk-grid-column-two-thirds">
 
         @forms.errorSummary(form.errors)
-        <h1 class="govuk-heading-l">@messages("view.write-response-text.heading")</h1>
+       
+        <div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="1000">
         @forms.formWithCSRF(action = postAction) {
-            @forms.inputCharacterCount(
-                form("responseText"),
-                hint = Some(messages("view.write-response-text.hint")),
-                label ="",
-                maxLength = Some(1000)
+            @forms.textarea(
+                form = form,
```

Comment:
Does this not break the welsh char count work that I did couple of sprints back?

Reply:
Thanks Sweta, I'm having a look at it now :)